docs/eog/case-studies-cw20-agenda.rst

```diff
@@ -0,0 +1,76 @@
+Participants suggested and signed up to the topics that they wanted to discuss in a spreadsheet (`example spreadsheet <https://doi.org/10.6084/m9.figshare.12498278>`_). 
+The topics that people signed up to were then assigned a Group ID - in this case using the letters of the alphabet. 
+The Zoom Host then created breakout rooms named Group A, Group B, etc. and assigned the people who signed up to those groups to the associated breakout rooms during the lunch break. 
+The groups were pointed to their discussion group reporting document via the “Groups” tab in the spreadsheet, in which they worked together on their `speed blog <http://bit.ly/ssi-speed-blogging>`_.
+
+Collaborative Ideas (Exploring)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `Collaborative Ideas <https://www.software.ac.uk/cw20/collaborative-ideas-and-hackday-ideas>`_ session was conducted in a similar way to the Discussion session except that the group assignments for the breakout rooms were randomly generated. 
+This time the Group IDs were kept as numerics, so the group that was assigned Breakout Room 1 retrieved their Collaborative Ideas document for Idea Group 1 in the associated spreadsheet (`example spreadsheet <https://doi.org/10.6084/m9.figshare.12500768>`_), and so on. 
+Normally, voting for the best Collaborative Ideas takes place on uCONFLY, but to streamline things we added a “Voting” tab to the spreadsheet and asked participants to simply add their names to the ideas that they wanted to vote for and tallied these up at the end.
+
+You can read more details about how we ran the Collaborative Ideas session in the Collaborations Workshop section of the `CSCCE's guide to Using virtual events to facilitate community building: event formats <https://doi.org/10.5281/zenodo.3934384>`_.
+
+Mini-workshops and Demos (Teaching)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We had 16 `Mini-workshops and Demos <https://www.software.ac.uk/cw20/mini-workshops-and-demo-sessions>`_ across four sessions. 
+We used a separate Zoom account for each of the four parallel workshops in each session, with an assigned Host (and Co-host for backup) for handling the logistics of the workshop (managing Zoom, recording the session, pointing to the relevant notes document) and helping the workshop facilitator with any technical issues. 
+We pointed people to the different Zoom room links in the day’s collaborative notes and in the Slack channel, and someone stayed in the main room to help people get to where they wanted to go. 
+A lesson learned is to provide guidance to workshop facilitators that the workshops should include participant interaction for at least half of the time, and not just one long presentation. 
+
+You can find the full collection of `CW20 infrastructure and documentation resources on Figshare <https://doi.org/10.6084/m9.figshare.c.5026400>`_.
```

Comment:
This sentence seems to float on it's own - it's not part of the Mini-workshops and Demos (Teaching) section - is it needed here? - if so perhaps give it it's own heading e.g. 'Infrastructure to help run the session' or put it at the end of the introductory paragraph on the page